chia/rpc/full_node_rpc_api.py

```diff
@@ -746,3 +751,59 @@ async def get_mempool_item_by_tx_id(self, request: Dict) -> EndpointResult:
             raise ValueError(f"Tx id 0x{tx_id.hex()} not in the mempool")
 
         return {"mempool_item": item}
+
+    async def get_fee_estimate(self, request: Dict) -> Dict[str, Any]:
+        if "spend_bundle" in request and "cost" in request:
+            raise ValueError("Request must contain ONLY 'spend_bundle' or 'cost'")
+        if "spend_bundle" not in request and "cost" not in request:
+            raise ValueError("Request must contain 'spend_bundle' or 'cost'")
+        if "target_times" not in request:
+            raise ValueError("Request must contain 'target_times' array")
+        if any([t < 0 for t in request["target_times"]]):
```

Comment:
I don't think we need the Mojos class, mainly because we have been very strict about only using mojos internally, and never XCH. I'd like to prevent that confusion ever happening, and I thought a new type around uint64 might do that.